src/BuildingRegistry.Grb.Abstractions/Migrations/BuildingGrbContextModelSnapshot.cs

```diff
@@ -112,6 +112,70 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("JobRecords", "BuildingRegistryGrb");
                 });
 
+            modelBuilder.Entity("BuildingRegistry.Grb.Abstractions.JobRecordArchive", b =>
+                {
+                    b.Property<long>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("bigint");
+
+                    SqlServerPropertyBuilderExtensions.UseIdentityColumn(b.Property<long>("Id"), 1L, 1);
```

Comment:
In the migration I don't see the Identity Column.
So since it's not mapped anymore, I don't think this file needs to change.
Maybe delete migration and create a new one? Copy the create table statements.